lib/ansible/parsing/vault/__init__.py

```diff
@@ -147,20 +147,11 @@ def is_encrypted_file(file_obj, start_pos=0, count=-1):
     current_position = file_obj.tell()
     try:
         file_obj.seek(start_pos)
-        vaulttext = file_obj.read(count)
```

Comment:
The count here is important.

Without it any file loaded by dataloader will be loaded entirely and entire contents ran through a to_bytes(to_text())